package/gluon-migrate-vpn/luasrc/lib/gluon/upgrade/499-migrate-vpn

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/lua
+
+local site = require 'gluon.site_config'
+local users = require 'gluon.users'
+local util = require 'gluon.util'
+
+local uci = require("simple-uci").cursor()
+
+
+function file_exists(name)
```

Comment:
Please use *nixio.fs.access* as in the other files instead of defining your own function.